.github/workflows/lighthouse-ci.yml

```diff
@@ -0,0 +1,14 @@
+name: CI
+on: [push]
+jobs:
+  lighthouseci:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18
+      - run: npm install && npm install -g @lhci/cli@0.11.x
```

Comment:
- we run `npm ci`
- is the global install necessary if we also have this in `devDependencies`?
- this version of @lhci/cli is different than what is declared in `package.json`. I suggest we use the `package.json` only, or perhaps https://github.com/treosh/lighthouse-ci-action - did you look at that at all?

Reply:
npm install was changed to npm ci